tests/test_utils/test_utils.py

```diff
@@ -5,21 +5,38 @@
 import openml
 from openml.testing import TestBase
 
+import pytest
 
-class OpenMLTaskTest(TestBase):
-    _multiprocess_can_split_ = True
```

Comment:
@mfeurer as far as I can tell this is a nosetest directive. Git blame says you added it, so maybe you know more?

Reply:
Yes, this is a nosetest directive and it basically allows nosetests to run tests in this class in parallel, which it would not do otherwise.

Reply:
But we don't use nosetest, so there is no reason to keep it around, right?

Reply:
Yes, indeed, no need to keep this around. I don't recall when this was necessary, maybe before we added Oslo to ensure that OpenML-Python is thread safe?